src/material/input/input.md

```diff
@@ -65,11 +65,19 @@ applies to all inputs. For convenience, `ShowOnDirtyErrorStateMatcher` is availa
 globally cause input errors to show when the input is dirty and invalid.
 
 ```ts
+// NgModule applications
```

Comment:
Can you split these into 2 separate code blocks? When its shown as one, I worry people might think they need to copy the entire snippet

Reply:
Good idea. Can it be two adjacent blocks like this

```ts
// NgModule applications
@NgModule({
  providers: [
    {provide: ErrorStateMatcher, useClass: ShowOnDirtyErrorStateMatcher}
  ]
})
```

```ts
// Standalone applications
export const appConfig: ApplicationConfig = {
  providers: [
    {provide: ErrorStateMatcher, useClass: ShowOnDirtyErrorStateMatcher}
  ]
};
```

or should the comments be pulled out into something like normal text above the blocks